### DISCUSSION TITLE
My KMP coil3("3.0.0-rc01") cannot load data from Firebase Storage

### DISCUSSION THREAD

**Discussion by nggdvu** [coil-kt/coil#2556] [Q&A]
Hi guys,

I'm doing a KMP project, and thankfully Coil 3 has started supporting multiplatform. After wandering around the Internet, I can see that the docs is being completed gradually. 

The point is, I couldn't load users' profile pictures from Firebase Storage inside androidMain. I don't know if it was my end, but afterward I changed the version to "2.7.0" then it worked like magic!

Any help would be appreciated. And keep up the lovely work!

**Comment by Teufelq** [comment]
did u resolve your issue? 'Cause i have same problem, but i even don't know how to upload my file into Firabase storage
